learn/cookbooks/laravel-multitenancy.mdx

```diff
@@ -0,0 +1,365 @@
+---
+title: Laravel multitenancy guide — Meilisearch documentation
+description: Learn how to implement secure, multitenant search in your Laravel applications.
+sidebarDepth: 3
+---
+
+# Laravel multitenancy guide
+
+This guide will walk you through implementing search in a multitenant Laravel application.
+
+## Requirements
+
+Before proceeding, you need a working Laravel 10 application with Laravel Scout installed and configured to use the `meilisearch` driver. Follow the official [Scout documentation](https://laravel.com/docs/10.x/scout) to get started.
+
+Additionally, you will need to have a Meilisearch server running and accessible from your Laravel application. If you don't have a Meilisearch server running, follow our [quick start guide](/learn/getting_started/cloud_quick_start) to get started with Meilisearch Cloud.
+
+<Capsule intent="tip">
+Prefer self-hosting? Read our [installation guide](/learn/getting_started/installation).
+</Capsule>
+
+To generate tenant tokens, you will need:
+- A search API key — available in your Meilisearch dashboard
+- A search API key UID — retrieve it using the [keys endpoints](/references/api/keys)
+
+## Use case
```

Comment:
We say "Use case", but the majority of this section seems to be about setting up Laravel models. I suggest we alter the heading to reflect that.

Perhaps something like this? I'm not super familiar with Laravel terminology, there might be a more idiomatic way to refer to it.

```suggestion
## Data models and relationships
```